### PR TITLE
feat(MenuToggle): consumed Penta updates

### LINKED ISSUE #9986: Consume Penta tokens: Menu toggle
Follow up to core PR https://github.com/patternfly/patternfly/pull/6177

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #9986
[Codemod](https://github.com/patternfly/pf-codemods/issues/584)

This also bumps the Core versions

**Additional issues**:

### CODE REVIEW

**Review by mattnolting — CHANGES_REQUESTED** [review]
Looks good, saw one bug

File: packages/react-core/src/components/MenuToggle/examples/MenuToggle.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { MenuToggle } from '@patternfly/react-core';
 
 ### Expanded toggle
 
-When a user selects a toggle, it becomes expanded and is styled with a blue underline. To flag expanded toggles, and apply expanded styling, use the `isExpanded` property . 
+When a user selects a toggle, it becomes expanded and is styled with a blue underline. To flag expanded toggles, and apply expanded styling, use the `isExpanded` property .
```
Comment (kmcfaul) [inline packages/react-core/src/components/MenuToggle/examples/MenuToggle.md:32]: Extra space at the end of the line

**Review by kmcfaul — APPROVED** [review]
lgtm just an extra space in the docs, but not a blocker

**Review by mattnolting — APPROVED** [review]
Awesome! Great work, thank you! LPTM

**Review by wise-king-sullyman — APPROVED** [review]

**Review by andrew-ronaldson — APPROVED** [review]
LGTM

**Pull request merged**